Data_Analysis_Markdown.Rmd

```diff
@@ -0,0 +1,503 @@
+                               scale(Dominican_Republic), svy)
+
+           
+#creating data frame of coefficients and CI's
+graph.coef <- summary(model1.graph)$coefficients[2:6, 1]
+model1.CI <- confint(model1.graph)
+model1.CIlower <- model1.CI[2:6 , 1]
+model1.CIupper <- model1.CI[2:6 , 2]
+
+labels = c("Skin Color","Parental Occupation", "Urban", "Age", "Female")
+
+frame <- data.frame(variable = labels,
+                    coefficient = graph.coef,
+                    ci.lower = model1.CIlower,
+                    ci.upper = model1.CIupper)
+
+#reorder variables
+frame$variable.order <- factor(frame$variable, levels = c("Female", 
+                                                          "Age", 
+                                                          "Urban", 
+                                                          "Parental Occupation", 
+                                                          "Skin Color"))
+
+
+#Create Figure 4
+fig4 <- ggplot(frame, aes(y = coefficient, x = variable.order)) + geom_point() +
+  geom_pointrange(aes(ymin = ci.lower,  ymax = ci.upper)) +
+  geom_hline(yintercept = 0, color ="dark green", size = 1) +
+  coord_flip() +
+  ylab("95 Percent C.I. (Design -Effects Based)") + xlab("") +
+  theme_classic() +
+  ggtitle("Effects of Skin Color and Other Factors on\n\n Educational Attainment in Select Latin American \n\nCountries") +
+  theme(plot.title = element_text(lineheight =.6, face ="bold", size = 12),
+        axis.text = element_text(colour = "black", face = "bold"), 
+        axis.title = element_text(face = "bold"))
+
+fig4.note <- arrangeGrob(fig4, 
+                         sub = textGrob("Source: Americas Barometer by LAPOP",
+                                        x = .1, hjust = 0, vjust = 0,
+                                        gp = gpar(fontface = "italic", fontsize = 10)))
+```
+
+***Here is our expansion:***
+
+We re-run a model sepratly for each country, so that we can see the disaggregated coefficients.
+
+**Here is our new graphic:** 
+
+![](Images/rep.png)
+
+**Here is our code:** 
+
+```{r message=FALSE}
+
+#extension:
+
+#prep
+country.names <- c("Brazil", 
+                   "Mexico", 
+                   "Guatemala", 
+                   "Colombia", 
+                   "Ecuador", 
+                   "Bolivia", 
+                   "Peru", 
+                   "Dominican Republic")
+
+country.graph <- list()
+country.graph.coef <- list()
+country.CI <- list()
+country.CIlower <- list()
+country.CIupper <- list()
+frame <- list()
+country.fig <- list()
+country.fig.note <- list()
+
+#loop through for each country
+
+for (name in country.names) {
+  country.graph[[name]] <- lm(scale(ed) ~ scale(colorr) + 
+                               scale(parent_occ) + 
+                               scale(Urban) + 
+                               scale(q2) + 
+                               scale(Female), data= filter(eight_pais, pais == name)) 
+  
```

Comment:
here for clarify, I might move the filter step outside of the lm step.  Right now this line of code is doing two pretty different things